.babelrc

```diff
@@ -1,8 +1,7 @@
 {
   "presets": ["es2015"],
-  "env": {
-    "test": {
-      "plugins": [ "istanbul" ]
-    }
-  }
+  "plugins": [
+    "transform-export-extensions",
+    "transform-object-rest-spread"
```

Comment:
This is useful for spreading objects, instead of using `Object.assign`

```js
const state = { foo: 'bar' }
const o = Object.assign({}, state, { foo: 'baz' })

const o = { ...state, { foo: 'baz' } }
```